srcpkgs/couchdb/template

```diff
@@ -1,40 +1,111 @@
 # Template file for 'couchdb'
 pkgname=couchdb
-version=1.7.1
-revision=2
+version=3.0.1
+revision=1
 wrksrc="apache-couchdb-${version}"
-build_style=gnu-configure
-conf_files="/etc/couchdb/default.ini /etc/couchdb/local.ini"
-hostmakedepends="erlang pkg-config"
-makedepends="libressl-devel icu-devel js-devel libcurl-devel"
+build_style=gnu-makefile
```

Comment:
Why change to gnu-makefile if you call `./configure` later? if it is not a `gnu-configure` than you can use `build_style=configure`

Reply:
i have update te pull request